src/quarot/nn/linear.py

```diff
@@ -0,0 +1,39 @@
+import torch
+
+from quarot.quant_utils import PackedQuantizedTensor
+
+
+class QuarotFP16Linear(torch.nn.Module):
```

Comment:
this is just emulating, right? the wight tensors are integers, but stored in fp16? can we have a docstring to explain this here, please.

Reply:
that's right, added short docstring here but will extend when quantization added, as here the weights aren't int yet.